pytorch_lightning/metrics/regression/mean_error.py

```diff
@@ -0,0 +1,120 @@
+import torch
```

Comment:
As stated earlier, I think it would be beneficially to keep each metric in a separate file

Reply:
Done!